Makefile.in

```diff
@@ -98,12 +98,10 @@ ifeq ($(shell echo "$(CHK_KVER)" | grep -Eq '.*\.el[8-9].*\.x86_64'; echo $$?),0
 PXDEFINES += -D__PX_BLKMQ__ -D__EL8__
 endif
 
-# CentOS Stream 9 specific detection
-ifeq ($(shell test -f "/etc/os-release"; echo $$?),0)
-ifeq ($(shell cat "/etc/os-release" | grep -q "CentOS Stream" && cat "/etc/os-release" | grep -q "VERSION_ID=\"9\""; echo $$?),0)
+# CentOS Stream 9-10 Specific kernel checks
+ifeq ($(shell echo "$(CHK_KVER)" | grep -Eq '.*\.el(9|10)\.x86_64'; echo $$?),0)
 PXDEFINES += -D__EL9_STREAM__
 endif
-endif
```

Comment:
the above changes are not part of this PR at all, they are unrelated, please remove.